docs/topics/mta-6-installing-web-console-on-openshift.adoc

```diff
@@ -226,23 +242,20 @@ The test results indicate that the minimum amount of memory for running {Product
 ** Installing the cluster monitoring tools requires an additional 5 GiB of memory.
 ====
 
-=== Eviction Threshold
-
-A node running applications requires sufficient reserve memory to operate successfully. Whenever an application uses more than its reserved memory allocation, it can trigger an out-of-memory event.
-
-The `--eviction-hard` flag directs the node to evict applications when available memory drops below an absolute value or percentage value.
-
-Set `--eviction-hard` flag along with memory reserves for the node.
+[id="eviction-threshold_{context}"]
+=== Eviction threshold
 
-The following example illustrates node memory allocation settings:
+Each node has a certain amount of memory allocated to it. Some of that memory is reserved for system services. The rest of the memory is intended for running pods. If the pods use more than their allocated amount of memory, it triggers an out-of-memory event.
```

Comment:
As we have previously mentioned `OOMKilled` error, also indicated by exit code 137, should we define what this means i.e that a container or pod was terminated because they used more memory than allowed. OOM stands for “Out Of Memory